setup.cfg

```diff
@@ -7,3 +7,5 @@ cover-erase=1
 with-doctest=0
 where=src
 
+[bdist_wheel]
+universal = 1
```

Comment:
I [recently submitted a change](http://zestreleaser.readthedocs.io/en/latest/changelog.html#id4) to `zest.releaser`: if `uneversal` is true, then so is `create-wheel`. So it's not necessary anymore 😄